src/ghvmctl-runner

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env bash
 
-export DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
-export WAYLAND_DISPLAY=wayland-0
-export DISPLAY=:0.0
-export HOME=/home/ubuntu
+XDG_RUNTIME_DIR=/run/user/"$(id -u)"
+export XDG_RUNTIME_DIR
```

Comment:
```suggestion
export XDG_RUNTIME_DIR="/run/user/$(id -u)"
```

Reply:
Indeed, but... 

```bash
❯ shellcheck /tmp/foo.sh

In /tmp/foo.sh line 3:
export FOO=$(bar)
       ^-^ SC2155 (warning): Declare and assign separately to avoid masking return values.

For more information:
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
```

Reply:
Ha, okay, let's leave it across two lines and just add the quotes?

Reply:
Ok, it's not technically needed but I agree it's a nicer style :)